site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js

```diff
@@ -24,5 +24,50 @@ describe('Test cases revolving around bulk uploading', () => {
             cy.get('#num_pages').should('be.visible');
             cy.get('#submission-mode-warning > .warning').should('have.text', 'Warning: Submitting files for bulk upload!');
         });
+
+        it(`${user} should be able to upload and split by QR code`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#use-qr').check();
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_qr.pdf', { force: true });
+            cy.get('#bulk-upload-submit').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload successful');
+        });
+
+        it(`${user} should be able to upload and split by page count`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#num_pages').type('2');
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_page_count.pdf', { force: true });
+            cy.get('#bulk-upload-submit').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload successful');
+        });
+
+        it(`${user} should be able to delete uploads`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_qr.pdf', { force: true });
+            cy.get('#bulk-upload-submit').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload successful');
+            cy.get('#bulk-upload-delete').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload deleted');
```

Comment:
Similarly here, we should verify not only that the 'Bulk upload deleted' banner appears, but that which we deleted has actually been deleted.